.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# **Problem:**
+
+- Describe the specific technical problem that this PR solves.
+- Include  details such as:
+    - How you reproduced the problem
+    - What code is affected
+    - Any relevant background information
+
+# **Solution:**
+
+- Describe the specific technical solution that this PR provides.
+- Include details such as:
```

Comment:
I think it would be great to also include how this PR might have an impact. For example, this change will improve the manifest generation speed by XX or this PR changes the behavior of manifest generation.

Reply:
Made a couple updates [here](https://github.com/Sage-Bionetworks-Workflows/py-dcqc/pull/58/commits/610e737c606f1c3a7c0e2f869065a0cbb245fae3)